Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphSchemaBuilder.php

```diff
@@ -42,7 +42,7 @@ private function createNodeTable(): Table
             DbalSchemaFactory::columnForDimensionSpacePointHash('origindimensionspacepointhash')->setNotnull(false),
             DbalSchemaFactory::columnForNodeTypeName('nodetypename'),
             (new Column('properties', Type::getType(Types::TEXT)))->setNotnull(true)->setCustomSchemaOption('collation', self::DEFAULT_TEXT_COLLATION),
-            (new Column('classification', Type::getType(Types::STRING)))->setLength(20)->setNotnull(true)->setCustomSchemaOption('charset', 'binary'),
+            (new Column('classification', Type::getType(Types::BINARY)))->setLength(20)->setNotnull(true),
```

Comment:
This change is not directly related but a follow-up to #4730 – without that change, `setup()` will always drop and re-add the column (and the status will reflect that)